.github/workflows/pr.yml

```diff
@@ -87,7 +87,7 @@ jobs:
     strategy:
       matrix:
         deployment:
-          - azure-1-staging
+          # - azure-1-staging
```

Comment:
Why comment out?

Reply:
Because parts of it are already commented out as it broke a test. In the future, we test against local Axiom anyways.